libbeat/scripts/Makefile

```diff
@@ -46,7 +46,7 @@ export PATH := ./bin:$(PATH)
 GOFILES = $(shell find . -type f -name '*.go' 2>/dev/null)
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "*/vendor/*" 2>/dev/null)
 GOFILES_ALL = $(GOFILES) $(shell find $(ES_BEATS) -type f -name '*.go' 2>/dev/null)
-GOPACKAGES_STRESSTESTS=$(shell find . -name '*.go' 2>/dev/null | xargs grep -l '\+build.*stresstest' | xargs -n1 dirname | uniq)
+GOPACKAGES_STRESSTESTS=$(shell find . -type d \( -name "stress" \) 2>/dev/null)
```

Comment:
What's the reason for this change?

Reply:
Because the original line doesn't work at all. And, if I understood that line correctly, the main idea is to get paths to folders with stress tests. For the particular beat, I found only one place with stress tests, this place is covered by the new line...

Reply:
https://github.com/elastic/beats/pull/21719 was the original change about 4 years ago.

As far as I see it's defined in [here](https://github.com/elastic/beats/blob/12b73b7543fe0bd805be25b08a3b2dec3cb47ae6/libbeat/Jenkinsfile.yml#L39-L41) for Jenkins and it [ran](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats/detail/main/2287/pipeline/2388) without running:

```
24-02-05T16:13:59.159Z] + make STRESS_TEST_OPTIONS=-timeout=20m -race -v -parallel 1 -C libbeat stress-tests
[2024-02-05T16:13:59.159Z] make: Entering directory '/var/lib/jenkins/workspace/main-2287-c122c791-d81b-466c-8ad7-4cbbc004111d/src/github.com/elastic/beats/libbeat'
[2024-02-05T16:13:59.587Z] dirname: missing operand
[2024-02-05T16:13:59.588Z] Try 'dirname --help' for more information.
[2024-02-05T16:13:59.746Z] dirname: missing operand
[2024-02-05T16:13:59.746Z] Try 'dirname --help' for more information.
[2024-02-05T16:13:59.746Z] if [ -n '' ]; then \
[2024-02-05T16:13:59.746Z] 	go test --tags=stresstest -timeout=20m -race -v -parallel 1 ; \
[2024-02-05T16:13:59.746Z] fi
```

I wonder whether this should be fixed in a different PR to avoid mixing concerns

Reply:
done, https://github.com/elastic/beats/pull/37873